python/cudf/cudf/tests/test_csv.py

```diff
@@ -1502,27 +1501,43 @@ def test_csv_writer_datetime_data(tmpdir):
     assert_eq(expect, got)
 
 
-@pytest.mark.parametrize("sep", [",", "|", " ", ";", np.str_(",")])
+@pytest.mark.parametrize("line_terminator", ["\r", "\n", "\t", np.str_("\n")])
+@pytest.mark.parametrize("sep", [",", "/", np.str_(",")])
+def test_csv_writer_terminator_sep(line_terminator, sep):
+    df = cudf.from_pandas(make_numpy_mixed_dataframe())
```

Comment:
It looks like the data that we're using that comes out of `make_numpy_mixed_dataframe()` is used for multiple tests, you might consider factoring it out and just generating it once, maybe as a pytest fixture

Reply:
Added fixtures for this. Can you please check if I did this correctly? I don't know much about fixtures.

Reply:
LGTM